EDDI/EDDI.cs

```diff
@@ -1572,42 +1572,50 @@ private bool eventLiftoff(LiftoffEvent theEvent)
 
         private bool eventMarket(MarketEvent theEvent)
         {
-            if (allowMarketUpdate)
+            // Don't proceed if we've already viewed the market while docked or when loading pre-existing logs
+            if (allowMarketUpdate && !theEvent.fromLoad)
```

Comment:
To simplify a bit, we could turn this `if` and return immediately.
E.g.
```
if( theEvent.fromLoad || ! allowMarketUpdate )
    return false;
....
```

Reply:
ok